ansible_runner/__main__.py

```diff
@@ -39,6 +39,7 @@
 
 from ansible_runner import run
 from ansible_runner import output
+from ansible_runner import cleanup
```

Comment:
Nit: combine multiple imports.
```suggestion
from ansible_runner import (
    cleanup,
    output,
    run,
)
```